app/controller/button/DrawingButtonController.js

```diff
@@ -44,13 +44,19 @@ Ext.define('CpsiMapview.controller.button.DrawingButtonController', {
      */
     snapInteraction: null,
 
+    /**
+     * If user has started to edit a line i.d. the first point of a line is already set
```

Comment:
Not sure what i.d. here means?

Reply:
I actually meant `i.e.`, but I simply replaced it with "this means". This might be better understandable.
Fixed with [8a1a771](https://github.com/compassinformatics/cpsi-mapview/pull/578/commits/8a1a7711b7d95d6f15d6ab4b2f9a073733558efd)

Reply:
I tried to google the equivalent of e.g. in German to see if it was that, should have thought of i.e. first!